code/modules/clothing/under/miscellaneous.dm

```diff
@@ -816,4 +816,31 @@
 /obj/item/clothing/under/explorer
 	desc = "A green uniform for operating in hazardous environments."
 	name = "explorer's jumpsuit"
-	icon_state = "explorer"
\ No newline at end of file
+	icon_state = "explorer"
+
+/obj/item/clothing/under/cohesion
+	name = "black cohesion suit"
+	desc = "A plain black cohesion suit intended to assist Prometheans in maintaining their form and prevent direct skin exposure."
+	icon_state = "cohesionsuit"
+	body_parts_covered = UPPER_TORSO|LOWER_TORSO|LEGS|ARMS

```

Comment:
Also, this line seems to be unnecessary, since this shares the same data as the default /clothing/under item.